src/api/XRInputPose.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { mat4_identity } from '../math';
+
+const PRIVATE = Symbol('@@webxr-polyfill/XRInputPose');
+
+const XRHandednessList = ['', 'left', 'right'];
+const XRPointerOriginList = ['head', 'hand', 'screen'];
+
+export default class XRInputPose {
```

Comment:
XRInputPose and XRInputSource should both be added to `./src/api/index.js` in order to be available to the global scope in feature detection scenarios (e.g. `if (window.XRInputSource) {}`)